app/src/main/kotlin/org/kotlinlsp/index/ScanFilesThread.kt

```diff
@@ -1,38 +1,46 @@
 package org.kotlinlsp.index
 
-import com.intellij.openapi.vfs.VirtualFile
 import org.kotlinlsp.analysis.modules.Module
 import org.kotlinlsp.analysis.modules.getModuleList
-import org.kotlinlsp.common.info
 import org.kotlinlsp.index.worker.WorkerThread
 import java.util.concurrent.atomic.AtomicBoolean
 
 class ScanFilesThread(
     private val worker: WorkerThread,
     private val rootModule: Module
-): Runnable {
+) : Runnable {
     private val shouldStop = AtomicBoolean(false)
 
     override fun run() {
         var gotFirstLibraryFile = false
 
         rootModule.getModuleList()
-            .sortedByDescending { it.isSourceModule }   // Index source files first so initial analysis takes less time
+            .sortedByDescending { it.isSourceModule }   // Scan source files first so initial analysis takes less time
             .map { it.computeFiles() }
             .flatten()
             .takeWhile { !shouldStop.get() }
             .forEach {
-                if(!it.url.endsWith(".kt") && !gotFirstLibraryFile) {
-                    worker.submitCommand(Command.SourceIndexingFinished)
+                if (!it.url.endsWith(".kt") && !gotFirstLibraryFile) {
+                    worker.submitCommand(Command.SourceScanningFinished)
                     gotFirstLibraryFile = true
                 }
-                val command = Command.IndexFile(it)
+                val command = Command.ScanFile(it)
                 worker.submitCommand(command)
             }
 
-        if(!gotFirstLibraryFile) worker.submitCommand(Command.SourceIndexingFinished)
+        if (!gotFirstLibraryFile) worker.submitCommand(Command.SourceScanningFinished)
 
-        worker.submitCommand(Command.IndexingFinished)
+        worker.submitCommand(Command.ScanningFinished)
```

Comment:
This command should be at the end, to signal that the whole background index is finished